pkg/service/cr_spec_builder.go

```diff
@@ -0,0 +1,90 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	olm "github.com/operator-framework/api/pkg/operators/v1alpha1"
+
+	"github.com/pkg/errors"
+	"github.com/tidwall/sjson"
+)
+
+// CRSpecBuilder provides all the functionalities to validate and build operands (operators) spec
+// based on schema available for them.
+type CRSpecBuilder struct {
```

Comment:
Shouldn't this be `CRDSpecBuilder` since we're trying to make spec for a CRD?

Reply:
same reason 
https://github.com/openshift/odo/pull/4704#discussion_r656492540

Reply:
https://github.com/openshift/odo/pull/4704#discussion_r657008906

Reply:
done